### ISSUE TITLE
Accessibility Bug: Add aria-controls to the accordion buttons on the A11y.html page

### ISSUE THREAD

**Issue by CorgiDev** [CorgiDev/CorgiDevSite#85]
The buttons on the accordion of the A11y.html page should have aria-controls set on each one with the value set to the id of the section beneath it.
